src/NuGet.Clients/NuGet.SolutionRestoreManager/VulnerablePackagesInfoBar.cs

```diff
@@ -21,19 +23,25 @@ namespace NuGet.SolutionRestoreManager
     [PartCreationPolicy(CreationPolicy.Shared)]
     public class VulnerablePackagesInfoBar : IVulnerabilitiesNotificationService, IVsInfoBarUIEvents
     {
+        private const string LogEntrySource = "NuGet Package Manager";
```

Comment:
I found there's an ILogger implementation that's a wrapper for the `ActivityLog` and has this source already defined. It would probably be better to use that to reduce duplicate code. Possibly in a follow-up PR? :-)